adopt-github-release/build.gradle

```diff
@@ -3,14 +3,14 @@ plugins {
     id 'application'
 }
 
-group 'net.adoptopenjdk.release'
+group 'net.adoptium.release'
 version '1.0.0-SNAPSHOT'
 
 repositories {
     mavenCentral()
 }
 application {
-    mainClassName = 'net.adoptopenjdk.release.UploadFiles'
```

Comment:
need to also adjust the package in the file at:
https://github.com/adoptium/github-release-scripts/blob/633ba620d3967652802c2e593a647c0dd0d05a28/adopt-github-release/src/main/groovy/net/adoptium/release/UploadFiles.groovy#L1